resources/lib/functions.py

```diff
@@ -588,7 +588,10 @@ def processDirectory(results, progress, params):
         name_format = settings.getSetting(name_format)
 
     dirItems = []
-    result = results.get("Items")
+    try:
+        result = results.get("Items")
```

Comment:
How about this instead of the try/except
result = results.get("Items")
if result is None:
   result = results